en/releases/README.md

```diff
@@ -1,8 +1,115 @@
 # Releases
 
-A list of PX4 release notes, they contain a list of the changes that went into each release, explaining the included features, bug fixes, deprecations and updates in detail.
+## Release Notes
+
+List of the changes that went into each release, explaining the included features, bug fixes, deprecations and updates in detail.
 
 * [v1.13](../releases/1.13.md)
 * [v1.12](../releases/1.12.md)
 
 The full archive of releases for the PX4 autopilot project can be found on [GitHub](https://github.com/PX4/PX4-Autopilot/releases)
+
+## Release Versioning
+
+PX4 uses a Semantic versioning (`Major.minor.patch[-beta#]`) system ([Wikipedia article](https://en.wikipedia.org/wiki/Software_versioning)).
+
+- Change in `Major`: Breaking changes
+- Change in `minor`: New non-breaking changes
+- Change in `patch`: All other non-breaking changes
+
+## Release Schedule
+
+PX4 follows the following release schedule:
+
+<!-- TO BE DEFINED -->
+
+- `Major` releases: By consensus among Maintainers & Community
+- `minor` releases: Roughly every 6 months
+- `patch` releases: When critical bug-fix / hardware support is needed
+- `beta` releases: Every 1 week during beta testing phase
+
+### Past Major/Minor releases
+
+There has been roughly 14 minor release in the past, with roughly 6 months ~ 1 year between new minor version releases. And there hasn't been any new major version releases yet.
+
+- [v1.13.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.13.0): June 22nd, 2022
+- [v1.12.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.12.0): July 11th, 2021
+- [v1.11.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.11.0): September 7th, 2020
+- [v1.10.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.10.0): December 17th, 2019
+- [v1.9.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.9.0): May 25, 2019
+- [v1.8.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.8.0): June 19th, 2018
+- [v1.7.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.7.0): December 15th, 2017
+- [v1.6.2](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.6.2): June 7th, 2017
+- [v1.5.1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.5.1): December 7th, 2016
+- [v1.4.1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.4.1): August 6th, 2016
+- [v1.3.1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.3.1): May 22nd, 2016
+- [v1.2.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.2.0): February 21st, 2016
+- [v1.1.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.1.0): December 24th, 2015
+- [v1.0.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.0.0): August 23rd, 2015
+
+## Release Procedure
+
+### Point release (Major/Minor change)
```

Comment:
Everywhere do first-letter capitalisation. It is doc-set wide. If you want to change it just for one doc. Don't.

```suggestion
### Point Release (Major/Minor Change)
```